classes/local/store/azure/file_system.php

```diff
@@ -37,4 +37,11 @@ protected function initialise_external_client($config) {
         $asclient = new client($config);
         return $asclient;
     }
+
+    /**
+     * @inheritdoc
+     */
+    public function supports_presigned_urls() {
+        return false;
```

Comment:
do we need this? why not just inherit from parent?

Reply:
updated